portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppServiceImpl.java

```diff
@@ -722,7 +726,8 @@ public FileEntry checkOutFileEntry(
 	@Override
 	public FileEntry copyFileEntry(
 			long fileEntryId, long destinationFolderId,
-			long destinationRepositoryId, ServiceContext serviceContext)
+			long destinationRepositoryId, List<Long> groupIds,
```

Comment:
For what I've seen, in most services long[] arrays are used when passing multiple group IDs. We may need to do the same here for consistency.